src/Symfony/Component/Validator/ConstraintValidator.php

```diff
@@ -86,6 +86,10 @@ protected function formatValue(mixed $value, int $format = 0): string
         }
 
         if ($value instanceof \UnitEnum) {
+            if (method_exists($value, 'from')) {
+                return $value->value;
+            }
```

Comment:
Why do we need this block? Can we keep logic concerning backed enums inside the validator that handles backed enums?